ion-transformation-storage-api/src/main/java/com/connexta/transformation/storage/api/TransformInfo.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.storage.api;
+
+import java.util.stream.Stream;
+
+public interface TransformInfo {
+
+  /**
```

Comment:
❗️ Additional information to be returned as part of this structure should be what is passed in parameter to the create. We should also see the start time, duration, aggregate status of the overall transform operation.